### PR TITLE
Remove some commented-out code in tests that's definitely not coming back

### PR DESCRIPTION
**Pull request by badboy**
### Pull Request checklist ###

- [ ] **Quality**: This PR builds and tests run cleanly
  - `make test` runs without emitting any warnings
  - `make lint` runs without emitting any errors
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Changelog**: This PR includes a changelog entry or an explanation of why it does not need one
  - Any breaking changes to language binding APIs are noted explicitly

### CODE REVIEW

File: glean-core/android/src/test/java/mozilla/telemetry/glean/private/LabeledMetricTypeTest.kt [hunk 1]
```diff
@@ -23,40 +17,17 @@ import org.junit.Assert.assertFalse
 import org.junit.Rule
 /*
 import java.util.UUID
```
Comment (Dexterp37) [inline glean-core/android/src/test/java/mozilla/telemetry/glean/private/LabeledMetricTypeTest.kt]: Let's remove this too. We're past the point in which we could simply do a file diff with a-c. If needed, we can add it back.

File: glean-core/android/src/test/java/mozilla/telemetry/glean/private/LabeledMetricTypeTest.kt [hunk 2]
```diff
@@ -260,45 +231,9 @@ class LabeledMetricTypeTest {
...
-        labeledTimespanMetric["label2"].start(this)
-        labeledTimespanMetric["label2"].stopAndSum(this)
-
-        assertTrue(labeledTimespanMetric["label1"].testHasValue())
-
-        collectAndCheckPingSchema(Pings.metrics).getJSONObject("metrics")!!
-        */
-    }
-
     @Ignore("UUID metric not yet implemented")
```
Comment (Dexterp37) [inline glean-core/android/src/test/java/mozilla/telemetry/glean/private/LabeledMetricTypeTest.kt]: AAARG, this fell through the cracks. Any chance you could file a bug about this? We should either remove it from a-c or implement it here.

File: glean-core/android/src/test/java/mozilla/telemetry/glean/private/LabeledMetricTypeTest.kt [hunk 3]
```diff
@@ -316,19 +251,15 @@ class LabeledMetricTypeTest {
...
-        UuidsStorageEngine.record(labeledUuidMetric["label2"], UUID.randomUUID())
+        // labeledUuidMetric["label1"].record(UUID.randomUUID())
+        // labeledUuidMetric["label2"].record(UUID.randomUUID())
 
-        collectAndCheckPingSchema(Pings.metrics).getJSONObject("metrics")!!
-        */
+        // collectAndCheckPingSchema(Pings.metrics).getJSONObject("metrics")!!
     }
 
     @Ignore("String List metric not yet implemented")
```
Comment (Dexterp37) [inline glean-core/android/src/test/java/mozilla/telemetry/glean/private/LabeledMetricTypeTest.kt]: As above, unrelated to this bug, but either we remove labelled string lists from a-c, or we add them here.

File: glean-core/android/src/test/java/mozilla/telemetry/glean/private/LabeledMetricTypeTest.kt (hunk 2 above)
Comment (badboy) [reply glean-core/android/src/test/java/mozilla/telemetry/glean/private/LabeledMetricTypeTest.kt]: Hm, I actually thought we _had_ an issue, that's why I left it in.
Labeled UUIDs and labeled string lists are not documented on our docs either, so I considered them "not implemented"

Comment (badboy) [reply glean-core/android/src/test/java/mozilla/telemetry/glean/private/LabeledMetricTypeTest.kt]: Filed https://bugzilla.mozilla.org/show_bug.cgi?id=1570303

File: glean-core/android/src/test/java/mozilla/telemetry/glean/private/LabeledMetricTypeTest.kt (hunk 3 above)
Comment (badboy) [reply glean-core/android/src/test/java/mozilla/telemetry/glean/private/LabeledMetricTypeTest.kt]: Filed https://bugzilla.mozilla.org/show_bug.cgi?id=1570304

**Pull request merged**